e2e/terraform/oebs-cloud.sh

```diff
@@ -99,8 +100,9 @@ function setup_local_env() {
         sleep 2
     else 
         echo "Missing terraform; Downloading and installing..."
-        wget "https://releases.hashicorp.com/terraform/0.10.0/terraform_0.10.0_linux_amd64.zip"
-        unzip terraform_0.10.0_linux_amd64.zip
+        terraform_url=$(curl https://releases.hashicorp.com/index.json | jq '{terraform}' | egrep "linux.*amd64" | sort --version-sort -r | egrep -v 'beta|rc' | head -1 | awk -F[\"] '{print $4}')
```

Comment:
Good one ! Any thoughts about egrep v/s grep -e , most linux flavours do support it though.

Reply:
egrep is equivalent to using "grep -E". Do you suggest I use "grep -E" instead?

Reply:
Right. grep -e would be safer, I suppose.

Reply:
Done, changed "egrep" to "grep -E".